fastplotlib/graphics/selectors/_rectangle.py

```diff
@@ -199,6 +199,43 @@ def __init__(
             edge.world.z = -0.5
             group.add(edge)
 
+        # vertices
+        top_left_vertex_data = (xmin, ymax, 1)
+        top_right_vertex_data = (xmax, ymax, 1)
+        bottom_left_vertex_data = (xmin, ymin, 1)
+        bottom_right_vertex_data = (xmax, ymin, 1)
+
+        top_left_vertex = pygfx.Points(
+            pygfx.Geometry(positions=[top_left_vertex_data], sizes=[edge_thickness]),
+            pygfx.PointsMaterial(size=edge_thickness, color=edge_color, size_space="world", size_mode="vertex"),
```

Comment:
let's do square points for resize-handlers, since that's commonly used in things like photoshop etc. circles tend to be rotate handles, but let's not worry about rotations here.

Reply:
and a different color for the vertices